testing/pavucontrol/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Fredrick Brennan <copypaste@kittens.ph>
+# Maintainer: Fredrick Brennan <copypaste@kittens.ph>
+pkgname=pavucontrol
+pkgver="20171007"
+pkgrel=1
+pkgdesc="A graphical interface for changing the volume of PulseAudio streams and other PulseAudio related tasks"
+url="https://freedesktop.org/software/pulseaudio/pavucontrol/"
+arch="all"
+license="GPL2+"
+depends="pulseaudio dbus gtkmm3 libsigc++ libcanberra"
+makedepends="automake autoconf gtkmm3-dev libsigc++-dev libcanberra-dev pulseaudio-dev"
+install=""
+subpackages="$pkgname-doc $pkgname-lang"
+_commit="335c26c57c18d95cc7d4ca693a75ef94fe919e1d"
```

Comment:
We should use release number instead of a random commit (which in practice means that you do the release and have to take responsibility and maintainership of it instead of letting upstream take that)

Reply:
How do I do a release on a project I'm not a maintainer of?

Reply:
Well, you can’t.

Reply:
I know, as referenced https://github.com/alpinelinux/aports/pull/2488#issuecomment-337126071. My port would not be the only port based on a commit.